src/screens/FeedScreen/Header.js

```diff
@@ -569,21 +569,21 @@ const styles = StyleSheet.create({
   feedDate: {
     fontFamily: fonts.inter[400],
     fontSize: 12,
-    color: COLORS.blackgrey,
+    color: COLORS.gray400,
     lineHeight: 18
   },
   feedDateLocation: {
     flex: 1,
     fontFamily: fonts.inter[400],
     fontSize: 12,
-    color: COLORS.blackgrey,
+    color: COLORS.gray400,
     lineHeight: 18
   },
   point: {
     width: 2,
     height: 2,
     borderRadius: 4,
-    backgroundColor: COLORS.blackgrey,
+    backgroundColor: COLORS.gray400,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color for `feedDate`, `feedDateLocation`, and `point` styles have been updated from `COLORS.blackgrey` to `COLORS.gray400`. As always, ensure these changes are consistent with your app's design guidelines and maintain good contrast for readability.
```diff
- color: COLORS.blackgrey,
+ color: COLORS.gray400,

- backgroundColor: COLORS.blackgrey,
+ backgroundColor: COLORS.gray400,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->